base_layer/wallet/src/output_manager_service/resources.rs

```diff
@@ -59,24 +57,25 @@ pub enum OutputManagerKeyManagerBranch {
     RecoveryByte,
 }
 
-impl Display for OutputManagerKeyManagerBranch {
-    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), Error> {
-        let response = match self {
-            OutputManagerKeyManagerBranch::Spend => "Spend",
-            OutputManagerKeyManagerBranch::SpendScript => "Script",
-            OutputManagerKeyManagerBranch::Coinbase => "Coinbase",
-            OutputManagerKeyManagerBranch::CoinbaseScript => "Coinbase_script",
-            OutputManagerKeyManagerBranch::RecoveryViewOnly => "Recovery_viewonly",
-            OutputManagerKeyManagerBranch::RecoveryBlinding => "Recovery_blinding",
-            OutputManagerKeyManagerBranch::RecoveryByte => "Recovery_byte",
-        };
-        fmt.write_str(response)
+/// Warning: Changing these strings will affect the backwards compatibility of the wallet with older databases or
+/// recovery.
+impl OutputManagerKeyManagerBranch {
+    pub fn get_branch_key(&self) -> String {
+        match self {
+            OutputManagerKeyManagerBranch::Spend => "".to_string(),
+            OutputManagerKeyManagerBranch::SpendScript => "script".to_string(),
+            OutputManagerKeyManagerBranch::Coinbase => "coinbase".to_string(),
+            OutputManagerKeyManagerBranch::CoinbaseScript => "coinbase_script".to_string(),
+            OutputManagerKeyManagerBranch::RecoveryViewOnly => "recovery_viewonly".to_string(),
+            OutputManagerKeyManagerBranch::RecoveryBlinding => "recovery_blinding".to_string(),
+            OutputManagerKeyManagerBranch::RecoveryByte => "Recovery_byte".to_string(),
```

Comment:
This doesn't look like it's been reverted

Reply:
yup that one is supposed to stay that way

Reply:
Ah probably wasn't there before and needs to stay the same now

Reply:
lol thanks github didnt refresh